src/olympia/reviewers/views.py

```diff
@@ -233,12 +232,8 @@ def dashboard(request):
     ):
         sections['Themes'] = [
             (
-                'New ({0})'.format(queue_counts['theme_nominated']),
-                reverse('reviewers.queue_theme_nominated'),
-            ),
-            (
-                'Updates ({0})'.format(queue_counts['theme_pending']),
-                reverse('reviewers.queue_theme_pending'),
+                'Manual Review ({0})'.format(queue_counts['theme']),
```

Comment:
Do we want to call it "Manual Review" when all themes are (right now) pre-review, so all manually reviewed? I don't have a great alternative title though... just "Review"?  "Awaiting Review"? 🤔

Reply:
`Awaiting Review` is probably better than `Manual Review` yeah.